gmachine.go

```diff
@@ -53,6 +53,10 @@ var PredefinedConstants = map[string]Word{
 	"STDERR":  PortStderr,
 }
 
+var Runes = map[string]Word{
+	"H": 72,
```

Comment:
I don't think we want to have to define all Unicode runes in a map, do we? I just checked and there are 120,803 of them!

Suppose we interpret any single character in single quotes as a rune value, and just use `rune(s)` to convert it. We could also treat any sequence of characters inside double quotes as a list of runes.

Reply:
> I don't think we want to have to define all Unicode runes in a map, do we? I just checked and there are 120,803 of them!
hahahah that would be funny... newbies.... hheh


> 
> Suppose we interpret any single character in single quotes as a rune value, and just use `rune(s)` to convert it. We could also treat any sequence of characters inside double quotes as a list of runes.
this sounds like a good idea.